src/runtime_src/core/common/query_requests.h

```diff
@@ -3685,9 +3685,10 @@ struct performance_mode : request
   enum class power_type
   {
     basic, // deafult
```

Comment:
Since we want to use the enum value as integer scalar, we should explicitly initialize them to integer values:
`basic = 0`